src/app/template.tsx

```diff
@@ -5,19 +5,22 @@ import Footer from "@/components/Footer";
 
 import "./global";
 
-export default function RootTemplate({
-  children,
-}: {
+interface RootTemplateProps {
   children: React.ReactNode;
-}) {
-  // const routeName = usePathname();
-  // const isLandingPagePath = routeName === "/";
+  announcementVisible?: boolean;
+}
 
+/**
+ * @notice The `announcementVisible` prop controls the visibility of the AnnouncementBar.
+ * @param {boolean} [announcementVisible=true] - If true, the AnnouncementBar will be hidden.
+ */
+export default function RootTemplate({
+  children,
+  announcementVisible = true,
```

Comment:
for positive logic,
`announcementVisible` should be `false` and 
```
if(!announcementVisible){
  return null
}
return <Box>...</Box>
```

Reply:
Thanks for pointing this out !